Aptoide/bds-sdk
# Appc 1147 wallet's custom store page

**What does this PR do?**

The call to the Wallet's store page was changed for this pull request. A verification is now being done to determine if Aptoide is installed on the user's device when he launches the billing flow
The wallet's store page is opened directly on Aptoide if it is installed or on any other market in case it's not
**Database changed?**

   No

**Where should the reviewer start?**

- [ ] GitHub/bds-sdk/android-appcoins- 
       billing/src/main/java/com/appcoins/sdk/billing/helpers/WalletUtils.java


**How should this be manually tested?**

  The toolbox should be ran and the launch billing flow started (the first wallet installation pop up from the poa must be disregarded). Test with and without Aptoide installed on the device and see if the wallet's page is opened directly on it or on the google play store in case aptoide ins't installed (the wallet package name must be "com.appcoins.wallet" for this test to take place)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1147](https://aptoide.atlassian.net/browse/APPC-1147)

**Questions:**

  How can we improve Aptoide's application starting time? It has a significant delay comparing to google play (opening the wallet's store page)

**Code Review Checklist**
